BUILD.md

```diff
@@ -213,16 +213,45 @@ Dockerfile instructions are available [here](./dockerfiles#tensorrt)
 
 #### NVIDIA Jetson TX1/TX2/Nano/Xavier
 
-1. Indicate CUDA compiler, or add its location to the PATH.
-Cmake can't automatically find the correct nvcc if it's not in the PATH.
-```
-export CUDACXX="/usr/local/cuda/bin/nvcc"
-```
-or:
-```
-export PATH="/usr/local/cuda/bin:${PATH}"
-```
-2. Follow instructions in [Docker README](./dockerfiles/README.md) to build the wheel file. 
+These instructions are for JetPack SDK 4.4.
+
+1. Clone the ONNX Runtime repo on the Jetson host
+
+    ```bash
+    git clone https://github.com/microsoft/onnxruntime
```

Comment:
should be git clone --recursive https://github.com/microsoft/onnxruntime

Reply:
Thanks @jywu-msft. Fixed.